### ISSUE TITLE
Add linting rule data from other available linters to existing violation data

### ISSUE THREAD

**Issue by MelSumner** [MelSumner/a11y-automation#209]
Add linting data from Deque's new [axe accessibility linter](https://marketplace.visualstudio.com/items?itemName=deque-systems.vscode-axe-linter) extension.

Information should be added to existing violation files first, if possible.

## Common
- [x] area-alt
- [ ] aria-command-name
- [ ] aria-dialog-name
- [ ] aria-input-field-name
- [ ] aria-meter-name
- [ ] aria-progressbar-name
- [ ] aria-toggle-field-name
- [ ] aria-tooltip-name
- [ ] aria-treeitem-name
- [ ] autocomplete-valid
- [ ] button-name
- [ ] frame-title
- [ ] image-alt
- [ ] input-button-name
- [ ] input-image-alt
- [ ] label
- [ ] link-name
- [ ] object-alt
- [ ] role-img-alt
- [ ] select-name
- [ ] svg-img-alt

## Markdown(MD)
- [ ] heading-order

## Vue
- [ ] aria-roles
- [ ] aria-valid-attr
- [ ] blink
- [ ] empty-heading
- [ ] interactive-supports-focus
- [ ] click-events-have-key-events
- [ ] mouse-events-have-key-events
- [ ] no-autofocus

## React
- [ ] aria-allowed-role
- [ ] aria-required-attr
- [ ] aria-roles
- [ ] aria-valid-attr
- [ ] aria-valid-attr-value
- [ ] blink
- [ ] empty-heading
- [ ] image-redundant-alt
- [ ] marquee
- [ ] scope-attr-valid
- [ ] tabindex
- [ ] no-interactive-element-to-noninteractive-role
- [ ] interactive-supports-focus
- [ ] click-events-have-key-events
- [ ] no-noninteractive-element-to-interactive-role
- [ ] no-static-element-interactions
- [ ] mouse-events-have-key-events
- [ ] no-autofocus
- [ ] anchor-is-valid

**Comment by MelSumner** [comment]
Closing this issue because not all of these rules have accompanying WCAG success criteria, so I will need to plan for how to handle those.
